acme/messages.py

```diff
@@ -182,15 +182,27 @@ def emails(self):
 
     @property
     def phone(self):
-        """Phone."""
-        assert len(self.phones) == 1
-        return self.phones[0]
+        """Phone.
```

Comment:
https://github.com/letsencrypt/letsencrypt/blob/master/letsencrypt/client.py#L126-128

I guess, I will have to fix that one as well...